app/src/main/java/com/cobresun/brun/pantsorshorts/repositories/impl/SharedPrefsUserDataRepository.java

```diff
@@ -94,6 +95,27 @@ public void writeLastFetchedTempLow(int temp) {
         editor.apply();
     }
 
+    @Override
+    public int[] readLastFetchedHourlyTemps() {
+        int defaultTemp = 10000;
+        int[] temps = new int[24 + MainActivityPresenter.HOURS_SPENT_OUT + 1];
```

Comment:
Change to static import of constant, its not great to import presenter into a repository, should be unknown to the `model` part of our app

Reply:
With the new adjustment to the algorithm based on the API return, we no longer need this import